tests/serac_solid_reuse_mesh.cpp

```diff
@@ -129,12 +125,12 @@ TEST(solid_solver, reuse_mesh)
   double dt = 1.0;
   solid_solver_3.advanceTimestep(dt);
 
-  double u_norm_3 = solid_solver_3.displacement().gridFunc().ComputeLpError(2.0, zerovec);
+  double u_norm_3 = solid_solver_3.displacement().norm();
   EXPECT_NEAR(0.0, u_norm_1 - u_norm_3, 0.001);
 
   solid_solver_3.resetToReferenceConfiguration();
-  EXPECT_NEAR(0.0, solid_solver_3.displacement().gridFunc().ComputeLpError(2.0, zerovec), 1.0e-8);
-  EXPECT_NEAR(0.0, solid_solver_3.velocity().gridFunc().ComputeLpError(2.0, zerovec), 1.0e-8);
+  EXPECT_NEAR(0.0, solid_solver_3.displacement().norm(), 1.0e-8);
+  EXPECT_NEAR(0.0, solid_solver_3.velocity().norm(), 1.0e-8);
```

Comment:
:+1: 
I prefer this inline implementation of the EXPECT_NEAR statement to some of the other uses we have that do something like
```cpp
// tmp's only purpose is to be put inside EXPECT_NEAR on the next statement,
// so it probably doesn't need to be an lvalue
double tmp = some_calculation(); 
EXPECT_NEAR(expected_value, tmp, tolerance);
```